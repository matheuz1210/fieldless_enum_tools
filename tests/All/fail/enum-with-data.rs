use fieldless_enum_tools::All;

#[derive(All)]
enum EnumWithData {
    A(u8),
    B { field1: u8, field2: u8 },
}

fn main() {}
