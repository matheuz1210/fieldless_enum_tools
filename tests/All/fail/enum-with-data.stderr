error: enum variant A has data on it! #[derive(All) can only be run on fieldless enums
 --> tests/All/fail/enum-with-data.rs:5:5
  |
5 |     A(u8),
  |     ^^^^^
